lightning/src/ln/channelmanager.rs

```diff
@@ -539,6 +540,32 @@ impl PendingOutboundPayment {
 	}
 }
 
+struct ExpandedPmtKey {
+	key_material: KeyMaterial,
+	metadata_key: [u8; 32],
+	ldk_pmt_hash_key: [u8; 32],
+	user_pmt_hash_key: [u8; 32],
+}
+
+impl Writeable for ExpandedPmtKey {
```

Comment:
Lets move away from storing key material in our own objects directly on disk - should be easy to just load this from the `KeysInterface` on restart, no?